modules/volume/rendering/renderabletimevaryingvolume.cpp

```diff
@@ -452,7 +451,7 @@ void RenderableTimeVaryingVolume::update(const UpdateData&) {
             _raycaster->setVolumeTexture(nullptr);
         }
         _raycaster->setStepSize(_stepSize);
-        _raycaster->setOpacity(opacity());
+        _raycaster->setBrightness(_brightness);
```

Comment:
I'd suggest to
```cpp
_raycaster->setBrightness(_brightness * opacity());
```
that wait we can use the `Fade` parameter that every Renderable has as well to fade out the volume rendering